bank_statement_import_payment_fix/security/ir.model.access.csv

```diff
@@ -0,0 +1 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
the security directory can be removed since they are no specific ACL defined in this addon